go.mod

```diff
@@ -12,3 +12,34 @@ require (
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 )
+
+require (
```

Comment:
Why do we have two `require` blocks now?

Reply:
This was automatically done by `go mod tidy`. Maybe this is something new in v1.17? You might want to check locally if your `go mod tidy` (v1.17) gives the same results. Note that the second `require` only contains indirect dependencies.

Reply:
Ok right, i get nearly the same result. Also with two `require` sections…